backend/src/main/java/com/google/models/Timestamp.java

```diff
@@ -68,7 +66,7 @@ public int getYear() {
 
   @Override
   public String toString() {
-    return year + "-" + month + "-" + day;
+    return date.toString();
```

Comment:
(related to the above comment in this file) why does this function now return just the local date to string?

Reply:
because this formats the string into yyyy-mm-dd, if i used the original way it could be yyyy-m-d or yyyy-mm-dd. its just so i don't have to manually write the `toString()` methods.